### PR TITLE
LRDOCS-3987 Marcos lexicon CSS updates

### PR DESCRIPTION
**Pull request by mwilliams2014**

### CODE REVIEW

File: develop/tutorials/articles/260-themes-and-layout-templates/04-lexicon/00-lexicon-and-themes-intro.markdown [hunk 1]
```diff
@@ -1,9 +1,9 @@
 # Lexicon CSS and Themes [](id=lexicon-and-themes)
 
-Lexicon CSS is an extension of Bootstrap's CSS Framework. Bootstrap is by far the 
-most popular CSS framework on the web. Also, it's open source, so anyone can use 
-it. Built with Sass, Lexicon CSS fills the front-end gaps between Bootstrap and the 
-specific needs of @product@.
+Lexicon's CSS is an extension of Bootstrap's CSS Framework. Bootstrap is by far 
+the most popular CSS framework on the web. Also, it's open source, so anyone can 
+use it. Built with Sass, Lexicon's CSS fills the front-end gaps between 
+Bootstrap and the specific needs of @product@.
```
Comment (blzaugg) [inline develop/tutorials/articles/260-themes-and-layout-templates/04-lexicon/00-lexicon-and-themes-intro.markdown]: Use:
```diff
- Lexicon's CSS
+ Lexicon
```

**Comment by blzaugg** [comment]
Lexicon's CSS feels odd. I think just Lexicon should be fine.

File: develop/tutorials/articles/260-themes-and-layout-templates/04-lexicon/00-lexicon-and-themes-intro.markdown (hunk 1 above)
Comment (blzaugg) [inline develop/tutorials/articles/260-themes-and-layout-templates/04-lexicon/00-lexicon-and-themes-intro.markdown]: Use just Lexicon here.

File: develop/tutorials/articles/260-themes-and-layout-templates/04-lexicon/01-importing-lexicon-css-in-a-theme.markdown [hunk 2]
```diff
@@ -1,8 +1,8 @@
 # Importing Lexicon CSS into a Theme [](id=importing-lexicon-css-into-a-theme)
 
-As mentioned before, Lexicon CSS fills the gaps between Bootstrap and the specific 
-needs of @product@. Bootstrap features have been extended to cover more 
-use cases. Here are some of the new components added by Lexicon CSS:
+As mentioned before, Lexicon's CSS fills the gaps between Bootstrap and the 
+specific needs of @product@. Bootstrap features have been extended to cover more 
+use cases. Here are some of the new components added by Lexicon's CSS:
```
Comment (blzaugg) [inline develop/tutorials/articles/260-themes-and-layout-templates/04-lexicon/01-importing-lexicon-css-in-a-theme.markdown]: Use just Lexicon here.

File: develop/tutorials/articles/260-themes-and-layout-templates/04-lexicon/01-importing-lexicon-css-in-a-theme.markdown [hunk 3]
```diff
@@ -29,15 +29,15 @@ consuming tasks such as these:
 - notification icons/messages
 - vertical alignment of content
 
-Next you can learn more about Lexicon's structure.
+Next you can learn more about Lexicon's CSS structure.
```
Comment (blzaugg) [inline develop/tutorials/articles/260-themes-and-layout-templates/04-lexicon/01-importing-lexicon-css-in-a-theme.markdown]: Use just Lexicon here.

File: develop/tutorials/articles/260-themes-and-layout-templates/04-lexicon/01-importing-lexicon-css-in-a-theme.markdown [hunk 4]
```diff
@@ -29,15 +29,15 @@ consuming tasks such as these:
...
 
-## Lexicon Structure CSS[](id=lexicon-css-structure)
+## Lexicon CSS Structure[](id=lexicon-css-structure)
```
Comment (blzaugg) [inline develop/tutorials/articles/260-themes-and-layout-templates/04-lexicon/01-importing-lexicon-css-in-a-theme.markdown]: Use just Lexicon here.

File: develop/tutorials/articles/260-themes-and-layout-templates/04-lexicon/01-importing-lexicon-css-in-a-theme.markdown [hunk 5]
```diff
@@ -29,15 +29,15 @@ consuming tasks such as these:
...
-Lexicon CSS is bundled with two sub-themes: Lexicon Base and Atlas. Lexicon Base is 
-@product@'s Bootstrap API extension. It is also the theme that is used in 
-@product@'s Styled Theme. It adds all the features and components you need and 
-inherits Bootstrap's styles. As a result, Lexicon Base is fully compatible with 
-third party themes that leverage Bootstrap's Sass variable API. As a best 
+Lexicon's CSS is bundled with two sub-themes: Lexicon Base and Atlas. Lexicon 
+Base is @product@'s Bootstrap API extension. It is also the theme that is used 
+in @product@'s Styled Theme. It adds all the features and components you need 
+and inherits Bootstrap's styles. As a result, Lexicon Base is fully compatible 
+with third party themes that leverage Bootstrap's Sass variable API. As a best
```
Comment (blzaugg) [inline develop/tutorials/articles/260-themes-and-layout-templates/04-lexicon/01-importing-lexicon-css-in-a-theme.markdown]: Use just Lexicon here.

File: develop/tutorials/articles/260-themes-and-layout-templates/04-lexicon/01-importing-lexicon-css-in-a-theme.markdown [hunk 6]
```diff
@@ -171,7 +171,7 @@ Follow these steps to add a third party theme:
 
 6.  Deploy your theme with `gulp deploy` (for legacy ant themes, use `ant deploy`)
 
-Now you know how to use Lexicon CSS in your theme!
+Now you know how to use Lexicon's CSS in your theme!
```
Comment (blzaugg) [inline develop/tutorials/articles/260-themes-and-layout-templates/04-lexicon/01-importing-lexicon-css-in-a-theme.markdown]: Use just Lexicon here.

**Comment by mwilliams2014** [comment]
OK. @marcoscv-work  can I ask why you submitted this change?

**Comment by mwilliams2014** [comment]
Closing for now until Marcos answers. Will resend if need be.

**Comment by marcoscv-work** [comment]
Hey guys,

Lexicon is a UX-experience-language https://lexicondesign.io/ and Lexicon CSS is the framework build over bootstrap to fit the gaps between bootstrap and Lexicon needs.

In other words, Lexicon CSS is the first version of the framework now called Clay, when we use "Lexicon" we are speaking about UX/Visual design and when we use "Lexicon CSS" we are speaking about the first version of our own CSS framework.

We are already using those terms:
https://dev.liferay.com/develop/tutorials/-/knowledge_base/7-0/importing-lexicon-css-into-a-theme

But if we are not consistent the customers could finish in a total confusion.

**Comment by mwilliams2014** [comment]
K. I'll remove my changes and resend the PR. Thanks @marcoscv-work !

**Comment by mwilliams2014** [comment]
Sent https://github.com/sez11a/liferay-docs/pull/2547

**Pull request closed without merging**